src/js/apps/clinicians/clinicians_state.js

```diff
@@ -0,0 +1,9 @@
+import Backbone from 'backbone';
+
+export default Backbone.Model.extend({
+  defaults() {
+    return {
+      searchQuery: '',
+    };
+  },
+});
```

Comment:
Does this `StateModel` need to be in its own file since it only contains the `searchQuery` value?

I contemplated placing it directly in the `clinicians-all_app.js` file as a `const` like this:

```js
// /src/js/apps/clinicians/clinicians-all_app.js

const StateModel = Backbone.Model.extend({
  defaults: {
    searchQuery: '',
  },
});
```

But I figured I'd follow the pattern we use for the other apps. And it sounds like we'll be adding more filter functionality to the clinicians page in the future, so it will probably need to be in its own file eventually anyways.

Reply:
Yeah I probably would have just popped this in the app, but it's fine either way.  _Although_ in this case.. I'm not even sure the default is useful or necessary.  It's more explicit as to what the state model is doing, but I'd suspect if no state model was defined, the default would still work ok.

Reply:
I removed the `src/js/apps/clinicians/clinicians_state.js` completely and everything works the same. So, appears to be no need for it.